ui/gulpfile.esm.js

```diff
@@ -21,9 +21,19 @@ task('swagger:watch', () =>
   watch(['../cmd/**/*.go', '../pkg/**/*.go'], series('swagger:generate'))
 )
 
-task('webpack:dev', shell.task('yarn react-app-rewired start'))
+task(
+  'webpack:dev',
```

Comment:
I was wondering why we need sentry in dev mode. It's very unstable when developing.

Reply:
Maybe we need a switch to control whether Sentry is on or not?

Reply:
@shhdgit What do you mean by being unstable？ Could you elaborate on this? Sending requests to sentry shouldn't hurt performance in a way you can feel.

It just tells us if it works well during dev so we can test all the features sentry provided. We can turn it off in dev tho, but it will bring a little bit more code complexity.

Reply:
My bad. Just curious about the usage of dev.
I mean there are too many useless error reports during development. May cause information overload.

Reply:
> may cause information overload

since we'll pass the environment property to sentry when init, it can be easily filtered.

![image](https://user-images.githubusercontent.com/18644268/115491173-15d2e600-a292-11eb-85c4-90cc84dbee41.png)

> maybe we can test all features when the switch on

I mean when setting up sentry (or changing configs afterwards) there are something I'd like to be certain that it works well before we publish and distribute the code to real users. Then it cannot be avoided to send requests to sentry during dev.

Reply:
Yes, I agree that we should test full function of Sentry before we publish. I mean like `dev-sentry` env will be better when testing sentry. Everyone in the dev env now reports to Sentry.

Reply:
OK, I think I get your point.

How about adding another `REACT_APP_SENTRY_ENABLED` env variable? It will be false by default in development and we can also manually change it (without commit it to git) if we want to test it in dev mode.

Reply:
Yes, I think it's great.

Reply:
fixed